templates/component/stateful.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+{{#ifIn 'toccoui' features}}
+import * as ToccoUi from 'tocco-ui'
+{{/ifIn}}
+
+class {{pascalCase componentName}} extends React.Component {
+  constructor(props) {
+    super()
+  }
+
+  render() {
+    const props = this.props
+
+    return (
+      <div>
+        <h1>{{titleCase componentName}}</h1>
+      </div>
+    )
+  }
+}
+
+{{pascalCase componentName}}.propTypes = {
+  val: React.PropTypes.func.isRequired
```

Comment:
I personally would leave the prop types empty. I don't like too much stuff in templates which has to be removed or changed anyway.